src/s1reader/s1_annotation.py

```diff
@@ -897,3 +912,120 @@ def _anx2height(cls, delta_anx):
             h_t += h_i * np.sin((i+1) * worb * delta_anx + phi[i])
 
         return h_t
+
+
+@dataclass
+class BurstExtendedCoeffs:
+    '''
+    Segments of FM rate / Doppler centroid polynomial coefficients.
+    For (linear) interpolation of FM rate / Doppler Centroid along azimuth.
+    To be used for calculating azimuth FM rate mismatch mitigation
+    '''
+
+    # FM rate
+    vec_aztime_fm_rate: np.ndarray
+    lut_coeff_fm_rate: np.ndarray
+    vec_tau0_fm_rate:np.ndarray
+
+    # Doppler centroid
+    vec_aztime_dc: np.ndarray
+    lut_coeff_dc: np.ndarray
+    vec_tau0_dc: np.ndarray
```

Comment:
1. It might be redundant, but I would prefer to keep the prefix (or postfix) to explicitly indicate the shape of the variable (i.e. 1d or 2d array), so that helps the users to avoid the confusion about all the variables about fm_rate_* or dc_* going around the code. It helped me to write this code, so I believe the others should as well.
3. I agree that lut can be misunderstood as LUT object in ISCE3. I'll change `lut` to `arr`
4. Good idea to make the variable names start with `fm_rate_` and `dc_`. I will make the changes. I think that would also help improve the readability in the `1.`